src/js/common/helpers/login-helpers.js

```diff
@@ -0,0 +1,34 @@
+// import moment from 'moment';
+import { commonStore } from '../store';
+
+import environment from './environment.js';
+import { updateLoggedInStatus, updateProfileField } from '../actions';
+
+export function handleVerify() {
+  const myStore = commonStore.getState();
+  const login = myStore.login;
+  const myVerifyUrl = login.loginUrl.third;
+  const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,top=50,left=500,width=500,height=750');
+  receiver.focus();
+}
+
+export function getUserData() {
+  fetch(`${environment.API_URL}/v0/user`, {
+    method: 'GET',
+    headers: new Headers({
+      Authorization: `Token token=${localStorage.userToken}`
+    })
+  }).then(response => {
+    return response.json();
+  }).then(json => {
+    const userData = json.data.attributes.profile;
+    commonStore.dispatch(updateProfileField('accountType', userData.loa.current));
```

Comment:
I wonder if it's worth it to have an action to update attributes en masse—probably not if we're only doing this type of updating in this component